com.unity.multiplayer.mlapi/Tests/Runtime/GlobalObjectIdHash/NetworkObjectGlobalObjectIdHashTests.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using NUnit.Framework;
+using UnityEditor.PackageManager;
+using UnityEditor.SceneManagement;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+
+namespace MLAPI.RuntimeTests
+{
+    public class NetworkObjectGlobalObjectIdHashTests
+    {
+        private Scene m_TestScene;
+
+        private void OnSceneLoaded(Scene scene, LoadSceneMode mode)
+        {
+            if (scene.name == nameof(NetworkObjectGlobalObjectIdHashTests))
+            {
+                m_TestScene = scene;
+            }
+        }
+
+        [UnitySetUp]
+        public IEnumerator Setup()
+        {
+            SceneManager.sceneLoaded += OnSceneLoaded;
+
+            var execAssembly = Assembly.GetExecutingAssembly();
+            var packagePath = PackageInfo.FindForAssembly(execAssembly).assetPath;
+            var scenePath = Path.Combine(packagePath, $"Tests/Runtime/GlobalObjectIdHash/{nameof(NetworkObjectGlobalObjectIdHashTests)}.unity");
+
+            yield return EditorSceneManager.LoadSceneAsyncInPlayMode(scenePath, new LoadSceneParameters(LoadSceneMode.Additive));
+        }
+
+        [UnityTearDown]
+        public IEnumerator Teardown()
+        {
+            SceneManager.sceneLoaded -= OnSceneLoaded;
+
+            if (m_TestScene.isLoaded)
+            {
+                yield return SceneManager.UnloadSceneAsync(m_TestScene);
```

Comment:
as we loaded our test scene additively above (and didn't change the main test scene), we also need to unload it to make it isolated from other tests running and keep the sandbox environment safe, untouched.